docs/rules/no-sequences.md

```diff
@@ -63,10 +63,46 @@ while ((val = foo(), val < 42));
 with ((doSomething(), val)) {}
 ```
 
+## Options
+
+This rule takes one option, an object, with the following properties:
+
+* `"allowInParentheses"`: If set to `false`, this rule allows to use expression sequences that are explicitly wrapped in parentheses. Default value is `true`.
```

Comment:
```suggestion
* `"allowInParentheses"`: If set to `true` (default), this rule allows to use expression sequences that are explicitly wrapped in parentheses.
```

The second part of the first sentence describes `true`, not `false`?

Reply:
My fault :) I wanted to express what needs to be done to change the default behavior. So I changed its second part ("If set to false, this rule disallows ...").